SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Shared.Targets/CrashesUtils.cs

```diff
@@ -8,6 +8,17 @@ internal static class CrashesUtils
     {
         internal static byte[] SerializeException(Exception exception)
         {
+            if (exception == null)
```

Comment:
How can it be null?

Reply:
With current usage - no way. This is just a guard

Reply:
`as` operator is used [here](https://github.com/Microsoft/AppCenter-SDK-DotNet/blob/724d830d80103939f236926a0e8e4a6069d4672b/SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.iOS/Crashes.cs#L102). So, it shouldn't be `null` but I would prefer that this check remains

Reply:
That does not help if it ever goes null as `GenerateiOSException` does not check for null. If we have any reason to think it might actually be null, add check there too.